src/index.js

```diff
@@ -171,24 +209,29 @@
             return false
         }
 
-        function init() {
-            const uuid = wc_infinitepay_params.uuid
-            const script = document.createElement('script')
-            script.setAttribute('src', 'https://authorizer-data.infinitepay.io/fp/tags.js?org_id=k8vif92e&session_id=cloudwalk' + uuid)
-            script.setAttribute('type', 'text/javascript')
-            document.head.appendChild(script)
-            const iFrame = document.createElement('div')
-            iFrame.setAttribute('id', 'iframe-cloudwatch-auth')
-            iFrame.innerHTML = '<noscript><iframe style="width: 100px; height: 100px; border: 0; position:absolute; top: -5000px;" src="https://authorizer-data.infinitepay.io/fp/tags?org_id=k8vif92e&session_id=cloudwalk' + uuid + '"></iframe></noscript>'
-            document.body.appendChild(iFrame)
-            const inputUUID = document.querySelector("#ip-uuid")
-            inputUUID.value = uuid
-        }
-
         $("form.woocommerce-checkout").on(
             "checkout_place_order",
             infinitePayFormHandler
         )
+
+        function init() {
+            if(!!window["IPay"]) return;
+
+            var head = document.getElementsByTagName("head")[0];
+            var script = document.createElement("script");
+            script.async = 1;
+            script.src = "https://ipayjs.infinitepay.io/production/ipay-latest.min.js";
```

Comment:
The src will be different for production and sandbox, right?

Reply:
Done 👍